mvc/src/main/java/webmvc/org/springframework/web/servlet/mvc/asis/ControllerHandlerAdapter.java

```diff
@@ -0,0 +1,23 @@
+package webmvc.org.springframework.web.servlet.mvc.asis;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import webmvc.org.springframework.web.servlet.ModelAndView;
+import webmvc.org.springframework.web.servlet.mvc.HandlerAdapter;
+import webmvc.org.springframework.web.servlet.view.JspView;
+
+public class ControllerHandlerAdapter implements HandlerAdapter {
+
+    @Override
+    public boolean supports(Object object) {
+        return object instanceof Controller;
+    }
+
+    @Override
+    public ModelAndView handle(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, Object handler) throws Exception {
+        Controller controller = (Controller) handler;
+        String viewName = controller.execute(httpServletRequest, httpServletResponse);
+        ModelAndView mav = new ModelAndView(new JspView(viewName));
+        return mav;
```

Comment:
```suggestion
return new ModelAndView(new JspView(viewName));
```

바로 리턴해도 될것 같아요!

별개로 질문드리자면, mav라는 네이밍은 ModelAndView를 축약한 네이밍인것 같은데, 흔히 이렇게 축약하여 사용하는 네이밍인가요??
(such as.. index -> idx, count -> cnt)
단순한 궁금증입니다!

Reply:
옛날에 디스패쳐 서블릿 코드를 까본기억이 있는데 그때도 mav라는 필드명을 쓰고있는것같더라고요 ,,! (아마 맞을겁니다)

그게 생각나서 저렇게 썼습니다..ㅋㅋ 흔히 쓰는지는 모르겠어요 ㅎㅅ

Reply:
아마 `HandlerAdapter`도 `ha`이렇게 썼던것같아요 ..ㅋㅋㅋ 알아보기가 힘들긴 하더라구요

Reply:
오.... 정말... 알아보기 힘든 네이밍이네요..ㅋㅋ
감사합니다!